_data/changelogs/component-header.yml

```diff
@@ -2,6 +2,16 @@ title: Header
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Reordered header items so that search bar appears last in mobile header.
+    summaryAdditional: Now, the header is in compliance with WCAG criterion 2.4.3 (Focus order) in mobile view. 
```

Comment:
**suggestion:** We should highlight that we removed the CSS order, instead of reordering the markup structure.

```suggestion
    summary: Removed the CSS order property from mobile header navigation.
    summaryAdditional: Now, the header is in compliance with WCAG criterion 2.4.3 (Focus order) in mobile view and all variants have consistent order. 
```